samples/get_vm_names.py

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/python
```

Comment:
The sample header comments need to be updated. Please take a look at filter_vm.py. Please update the year to 2017 too.